src/igvfd/tests/test_schema_rodent_donor.py

```diff
@@ -151,3 +151,18 @@ def test_taxa(award, lab, testapp):
             'strain': 'B6'
         }, expect_errors=True)
     assert res.status_code == 422
+
+
+def test_rodent_traits(rodent_donor, phenotype_term_myocardial_infarction, phenotype_term_alzheimers, testapp):
+    res = testapp.patch_json(
+        rodent_donor['@id'],
+        {'traits':
+            [phenotype_term_myocardial_infarction['@id'], phenotype_term_alzheimers['@id']]
+         })
+    assert res.status_code == 200
+    res = testapp.patch_json(
+        rodent_donor['@id'],
+        {'traits':
+            ['HP:0001658', 'DOID:10652']
```

Comment:
I am not sure what is the purpose of this test? if it is to test for uniqueness I don't think it works as you expect it?